### ISSUE TITLE
wevote.me/TwitterHandle - Return Candidate or Organization

### ISSUE THREAD

**Issue by DaleMcGrew** [wevote/WebApp#261]
I have created an API Endpoint https://api.wevoteusa.org/apis/v1/docs/twitterIdentityRetrieve/ 
If you pass it a twitter handle, it returns the "kind_of_owner" (CANDIDATE or ORGANIZATION) and the "we_vote_id" of the owner of the twitter handle.

In "src/js/routes/NotFound.jsx" (in Pull request #260) I show how to display a Candidate card once we have the we_vote_id.

The task at hand is to take the "TwitterHandle" from the URL, look it up via twitterIdentityRetrieve, and display the corresponding candidate or organization.

**Comment by DaleMcGrew** [comment]
Lisa got this working! (on development machines) @josephevans is working on fixing the routing on the live servers.

**Comment by lisamcho** [comment]
@DaleMcGrew What is wrong with the routing on live servers?

**Comment by DaleMcGrew** [comment]
On my development machine, http://localhost:3000/defenders (an organization) routes correctly to the org page. But on the live server we get a 404 from the ngnx load balancer.

**Comment by DaleMcGrew** [comment]
Hi @josephevans here is more details: we just implemented a feature where you can take the twitter handle, and use it to find either an organization or candidate on We Vote. In React this is implemented by taking any route that doesn't exist and looking it up as if it is a Twitter handle.

https://twitter.com/defenders
becomes:
https://wevote.me/defenders

Currently when I try to visit any url like "https://wevote.me/defenders" and I get an nginx 404 error. Could you please take a look when you have a moment?

**Comment by josephevans** [comment]
I modified the nginx config to route everything unknown to index.html, so it seems to be working now. Can someone verify and close this issue if everything seems to work?

Explanation: Previously, I had a list of URL paths that would be rewritten to index.html, but since there are potentially many more, I changed the logic to always redirect unless it's a known fonts/css/js/favicon/apple image link paths.
